prac_06/guitar_test.py

```diff
@@ -0,0 +1,36 @@
+"""
+CP1404 Prac06 - Guitar test
+Sam Butters
+Time started: 13:48
+Time finished:
+"""
+
+from prac_06.guitar import Guitar
+
+# guitar = Guitar("Gibson L-5 CES", 1922, 16035.40)
+# current_year = 2024
+#
+# print(guitar, Guitar.get_age(current_year), Guitar.is_vintage(current_year))
+
+
+CURRENT_YEAR = 2017
```

Comment:
I'm curious why you add the current year as 2017, for testing purposes like this i this making the CURRENT_YEAR a nice round number like 2000 would be great to make the math easy.